platform/testing/end-to-end/cypress-best-practices-on-vsp.md

```diff
@@ -335,105 +300,131 @@ https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/p
+    devicesWithViewport:
+      'iPhone XS Max, iPhone XR, iPhone 11, iPhone 11 Pro Max',
     percentTraffic: 9.75,
     percentTrafficPeriod: 'October, 2020',
+    viewportPreset: 'va-top-mobile-1',
     width: 414,
     height: 896,
   },
   {
-    name: 'iPhone 6, iPhone 6s, iPhone 7, iPhone 8',
+    list: 'VA Top Mobile Viewports',
+    rank: 2,
+    devicesWithViewport: 'iPhone 6, iPhone 6s, iPhone 7, iPhone 8',
     percentTraffic: 5.52,
     percentTrafficPeriod: 'October, 2020',
+    viewportPreset: 'va-top-mobile-2',
     width: 375,
     height: 667,
   },
   {
-    name: 'iPhone X, iPhone XS, iPhone 11 Pro',
+    list: 'VA Top Mobile Viewports',
+    rank: 3,
+    devicesWithViewport: 'iPhone X, iPhone XS, iPhone 11 Pro',
     percentTraffic: 5.5,
     percentTrafficPeriod: 'October, 2020',
+    viewportPreset: 'va-top-mobile-3',
     width: 375,
     height: 812,
   },
   {
-    name: 'iPhone 6 Plus, iPhone 6s Plus, iPhone 7 Plus, iPhone 8 Plus',
+    list: 'VA Top Mobile Viewports',
+    rank: 4,
+    devicesWithViewport:
+      'iPhone 6 Plus, iPhone 6s Plus, iPhone 7 Plus, iPhone 8 Plus',
     percentTraffic: 3.25,
     percentTrafficPeriod: 'October, 2020',
+    viewportPreset: 'va-top-mobile-4',
     width: 414,
     height: 736,
   },
   {
-    name: 'iPhone 5, iPhone 5s, iPhone 5c, iPhone SE 1st gen',
+    list: 'VA Top Mobile Viewports',
+    rank: 5,
+    devicesWithViewport: 'iPhone 5, iPhone 5s, iPhone 5c, iPhone SE 1st gen',
     percentTraffic: 1.49,
     percentTrafficPeriod: 'October, 2020',
+    viewportPreset: 'va-top-mobile-5',
     width: 360,
     height: 640,
   },
 ];
 ```
 
-To test against each of the viewports in the array, simply iterate through the array using `.forEach()` in the callback of an `it` function call, like so:
+To test against each of the viewports in the array, simply iterate through it using `.forEach()` in the callback of an `it` function call, like so:
 
 ```javascript
-it.only('should render in mobile layouts and tabs actions work', () => {
-  Cypress.env().vaTop5MobileViewports.forEach((viewportData) => {
+it('should render in mobile layouts', () => {
+  Cypress.env().vaTopMobileViewports.forEach((viewportData) => {
     const {
-      name,
-      width,
-      height,
+      list,
+      rank,
+      devicesWithViewport,
       percentTraffic,
       percentTrafficPeriod,
+      viewportPreset
+      width,
+      height,
     } = viewportData;
 
-    cy.log(`Device Group: ${name}`);
+    cy.log(`Viewport list: ${list}`);
+    cy.log(`Viewport rank: ${rank}`);
+    cy.log(`Viewport width x height: ${width} x ${height}`);
+    cy.log(`Devices with viewport: ${devicesWithViewport}`);
     cy.log(
       `% traffic for the month of ${percentTrafficPeriod}: ${percentTraffic}%`
     );
 
-    cy.visit('/find-locations');
-    cy.injectAxe();
-    cy.viewport(width, height);
-    cy.checkSearch();
+    cy.visit('/page');
+    cy.viewportPreset(viewportPreset);
+    // code omitted
```

Comment:
Just checking in on this // code omitted comment.

Reply:
Good catch. For clarity, perhaps it should say `// test code omitted` or something like that?